### DISCUSSION TITLE
Poetry lock and poetry add for pip installed packages

### DISCUSSION THREAD

**Discussion by Haislich** [python-poetry/poetry#8561] [General]
I use poetry in a computer that works behind a proxy.
As of now as far as I understand there is no option for proxies in poetry add and poetry lock.
If I install the dipendencies through pip, how can I then use poetry add and poetry lock?

**Comment by clintonroy** [comment]
The standard HTTP_PROXY and HTTPS_PROXY environment variables should work.
